code/src/nuvla/job/actions/utils/resource_log_fetch.py

```diff
@@ -40,6 +40,8 @@ def __init__(self, _, job):
         self.resource_log_id = self.job['target-resource']['href']
         self.resource_log = self.get_resource_log(self.resource_log_id)
         self.resource_log_parent = self.resource_log['parent']
+        self.api_dpl = Deployment(self.api)
+        self.deployment = self.api_dpl.get(self.resource_log_parent)
```

Comment:
I don't understand what this code is supposed to do.

Reply:
Removed the code and the Deployment import statement